### PR TITLE
Update relay lists to match what is on DoH & DoT

### PR DESCRIPTION
**Pull request by jbjorkang**
This PR ensures that our DNS over HTTPS and DNS over TLS lists are identical to the relay lists. Tracker blocking is not included on the non-relays, so that is as it was.

### CODE REVIEW

File: inventory/group_vars/all.yml [hunk 1]
```diff
@@ -8,11 +7,1913 @@ dns_replace_address:
...
+      9377if.com
+      93vitui.com
+      95105012.com
+      95p2.com
+      991pao.com
+      99tuan.com.cn
+      9h6qq.cn
+      9pb0h.cn
+      :34.66.206.188
+      :35.224.227.218
```
Comment (oskaralmlov) [inline inventory/group_vars/all.yml:205]: What are these? They look funky and will probably break BIND in later stages.

File: lists/relay/adblock/adaway [hunk 2]
```diff
@@ -0,0 +1,10938 @@
+
+  localhost
+::1  localhost
```
Comment (oskaralmlov) [inline lists/relay/adblock/adaway:3]: Seems weird to have localhost in our blocklist?

Comment (jbjorkang) [reply lists/relay/adblock/adaway]: See latest PR. Have fixed.

File: inventory/group_vars/all.yml [hunk 3]
```diff
@@ -8,11 +7,1913 @@ dns_replace_address:
...
+
 # List of extra domains to block, in addition to what we have listed
 # in `dns_adblock_lists`
 # Example:
 # dns_adblock_lists_custom:
 #   - type: adblock
 #     content: |
 #       ...
-dns_adblock_lists_custom: []
+dns_adblock_lists_custom:
```
Comment (Rizzly) [inline inventory/group_vars/all.yml:39]: Would be awesome if we could have a comment on where these came from as well, for posterity.

File: inventory/group_vars/all.yml (hunk 1 above)
Comment (jbjorkang) [reply inventory/group_vars/all.yml:207]: Added by Support & Operations for the DoH server originally.

File: inventory/group_vars/all.yml (hunk 3 above)
Comment (jbjorkang) [reply inventory/group_vars/all.yml:39]: I'll verify exactly where they came from, but I suspect they're custom additions to help with 'generic' website adblocking via Support & Operations.

**Review by Rizzly — APPROVED** [review]

**Comment by jbjorkang** [comment]
Merged into canary branch.

**Pull request closed without merging**